### ISSUE TITLE
SNPs not read for hg19

### ISSUE THREAD

**Issue by geertvandeweyer** [shendurelab/MIPGEN#10]
Hi, 

mipgen does not read in snps for hg19-style dbsnp files (with the 'chr' notation). BED files of targets were correctly read, and results were provided, so it's just in the reading of the snp file, or fetching by tabix. 

Best, 

Geert

**Comment by augustboyle** [comment]
Hello,

Still having issues? Make sure that the variants are in proper vcf format and that the chromosome names in your file match the chromosome names in your reference. Those are definitely the most common issues.

Evan

**Comment by janrehker** [comment]
Hi,

Not sure whether this is still causing trouble.. I just ran into a similar situation with mipgen. It seems that tabix was stumbling over the SNP file I got from dbSNP. It seems the issue was that the vcf was compressed in a way that is incompatible with tabix. For me the solution was to decompress the file with gunzip *.vcf.gz, then recompressing it with bgzip *.vcf 
To check if tabix was OK with that, I reindexed it with tabix -p vcf *.vcf.gz
After that mipgen loaded the SNPs and created some alternative MIPs.
